frontend/src/app/App.tsx

```diff
@@ -26,15 +25,13 @@ const App: React.FC<AppProps> = ({ detectUser }) => {
   };
 
   return (
-    <QuickStarts>
```

Comment:
If you don't need to be able to have the quick starts drawer open while navigating through the different pages in ODH then this is fine

Reply:
I don't think we want them open on non-quick start launch pages. The explore page has its own side drawer to show getting started content.